project/assets/main/puzzle/levels/tutorial-basics-4.json

```diff
@@ -100,14 +100,15 @@
     "top_out 999999"
   ],
   "other": [
-    "start_level tutorial_beginner_0",
+    "start_level tutorial_basics_0",
     "after_tutorial"
   ],
   "piece_types": [
     "start_piece_q", "start_piece_l"
   ],
   "rank": [
-    "skip_results"
+    "skip_results",
+    "unranked"
```

Comment:
Describe this change in the commit comment.

Reply:
Expanded commit comment, `Changed tutorials to unranked to avoid an unusual case where...`